flang/lib/Lower/OpenACC.cpp

```diff
@@ -31,27 +31,83 @@ GetDesignatorNameIfDataRef(const Fortran::parser::Designator &designator) {
 
 static void genObjectList(const Fortran::parser::AccObjectList &objectList,
                           Fortran::lower::AbstractConverter &converter,
-                          std::int32_t &objectsCount,
-                          SmallVector<Value, 8> &operands) {
+                          SmallVector<Value, 2> &operands) {
```

Comment:
Do we expect that we'll almost never have more than 2 operands?

We should consider using SmallVectorImpl as the interface though. Like here: https://github.com/llvm/llvm-project/blob/master/mlir/include/mlir/IR/FunctionImplementation.h#L57-L62

Reply:
Well I guess we will have cases where we have more for sure. I'm fine with updating it with the `SmallVectorImpl`.